mmoda/Dockerfile

```diff
@@ -0,0 +1,50 @@
+# For finding latest versions of the base image see
```

Comment:
We have since changed the Dockerfile to use a 2-stage build to try to improve caching and take advantage of parallel builds - you might want to do the same? See https://github.com/SwissDataScienceCenter/renku-project-template/blob/master/python-minimal/Dockerfile

Reply:
Thanks for the suggestion, it is following the same approach now